js/privatebin.js

```diff
@@ -2763,8 +2787,7 @@ jQuery.PrivateBin = (function($, RawDeflate) {
             // extract mediaType
             const mediaType = attachmentData.substring(5, mediaTypeEnd);
             // extract data and convert to binary
-            const rawData = attachmentData.substring(base64Start);
-            const decodedData = rawData.length > 0 ? atob(rawData) : '';
```

Comment:
This line probably got accidentally reverted. I had added the length check recently to address the bug reported in #663. I am fine to remove the `rawData` assignment, though, as keeping this in a variable only supports readability, but it isn't reused further on.